osu.Game.Rulesets.Taiko/UI/DrawableTaikoMascot.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Audio.Track;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Graphics.Containers;
+
+namespace osu.Game.Rulesets.Taiko.UI
+{
+    public sealed class DrawableTaikoMascot : BeatSyncedContainer
```

Comment:
Any reason why this is `sealed`? Not sealing could actually help in tests here (to which I'll get in another comment).